docs/research/bats-core.md

```
@@ -0,0 +1,133 @@
+# Research: bats-core v1.12/v1.13 Changes and run Behavior
+
+**Project:** claude-config
+**Last Updated:** 2026-04-11
+
+## Update Log
+| Date | Summary |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)